packages/wdio-config/tests/utils.test.ts

```diff
@@ -64,4 +71,23 @@ describe('utils', () => {
             expect(isCloudCapability({})).toBe(false)
         })
     })
+
+    describe('ensureAbsolutePathForSpecs', () => {
+        it('should properly add absolute path for spec with absolute path and line number', () => {
+            expect(ensureAbsolutePathForSpecs([FIXTURES_CUCUMBER_FEATURE_A_LINE_2])).toContainEqual(FIXTURES_CUCUMBER_FEATURE_A_LINE_2)
+        })
+
+        it('should properly add absolute path for spec with relative path and line number', () => {
+            expect(ensureAbsolutePathForSpecs([FIXTURES_CUCUMBER_FEATURE_A_LINE_2_RELATIVE_PATH])).toContainEqual(FIXTURES_CUCUMBER_FEATURE_A_LINE_2)
+        })
+
+        it('should properly add absolute path for spec with absolute path and line numbers', () => {
+            expect(ensureAbsolutePathForSpecs([FIXTURES_CUCUMBER_FEATURE_A_LINE_2_AND_12])).toContainEqual(FIXTURES_CUCUMBER_FEATURE_A_LINE_2_AND_12)
```

Comment:
you are asserting `FIXTURES_CUCUMBER_FEATURE_A_LINE_2_AND_12` with `FIXTURES_CUCUMBER_FEATURE_A_LINE_2_AND_12` which is like asserting true with true

Reply:
I am making sure that in case if the absolute path is sent , then the method should return it without changing anything